plugins/PRAplugin/src/MCSSolver.py

```diff
@@ -0,0 +1,144 @@
+# Copyright 2017 Battelle Energy Alliance, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Created on June 24, 2020
+
+@author: mandd
+"""
+
+#External Modules---------------------------------------------------------------
+import numpy as np
+import itertools
+import math
+#External Modules End-----------------------------------------------------------
+
+#Internal Modules---------------------------------------------------------------
+from PluginsBaseClasses.ExternalModelPluginBase import ExternalModelPluginBase
+from PostProcessors.MCSimporter import mcsReader
+#Internal Modules End-----------------------------------------------------------
+
+class MCSSolver(ExternalModelPluginBase):
+  """
+    This class is designed to create a MCS solver model
+  """
+  def __init__(self):
+    """
+      Constructor
+      @ In, None
+      @ Out, None
+    """
+    ExternalModelPluginBase.__init__(self)
+
+  def initialize(self, container, runInfoDict, inputFiles):
```

Comment:
since it is not used,
you can just remove the initialize

Reply:
Apparently this method is required, if not present this model will to be recognized